_docs/manage/cluster.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: guide
+title: StorageOS Docs - Cluster
+anchor: manage
+module: manage/cluster
+---
+
+# Cluster overview
+
+A StorageOS cluster needs to know the exact cluster size and peers to connect to
+during start up. All the nodes need to bind to the correct addresses in order to
```

Comment:
I've moved that line into the commented section and added the `storageos node ls` command.